src/formatHit.js

```diff
@@ -2,6 +2,9 @@ import findCountryCode from './findCountryCode.js';
 import findType from './findType.js';
 
 function getBestHighlightedForm(highlightedValues) {
+  if (highlightedValues.length === 0) {
```

Comment:
Ok, I would remove it still. I feel like everytime we do this, it leads to defensive programming, dead code + FUD (= we will keep this code, just in case)